tests/serializers/tests.py

```diff
@@ -340,6 +341,13 @@ def test_serialize_proxy_model(self):
         self.assertEqual(base_data, proxy_data.replace('proxy', ''))
         self.assertEqual(base_data, proxy_proxy_data.replace('proxy', ''))
 
+    def test_serialize_inheritance(self):
+        child_object = Child.objects.create(parent_data=2, child_data=3)
+        child_object.parent_m2m.add(Parent.objects.create(parent_data=4))
+        child_data = serializers.serialize(self.serializer_name, [child_object])
+        self.assertNotIn('parent_data', child_data)
+        self.assertNotIn('parent_m2m', child_data)
```

Comment:
`assertNotIn` may pass from many reasons. I think it is better to check field value with `self._get_field_values()` hook, e.g.
```python
self.assertEqual(self._get_field_values(child_data, 'parent_m2m'), [])
```